packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -1244,18 +1244,19 @@ export class CallContext {
     // if inProgressRealTimeTexts is an array
     if (inProgressRealTimeTexts && Array.isArray(inProgressRealTimeTexts)) {
       // find the in progress real time text that has not been updated for 5 seconds
-      inProgressRealTimeTexts.forEach((realTimeText, index) => {
+      for (let i = inProgressRealTimeTexts.length - 1; i >= 0; i--) {
+        const realTimeText = inProgressRealTimeTexts[i] as RealTimeTextInfo;
```

Comment:
remove `as RealTimeTextInfo`, type should be inferred

Reply:
Updated